From c0e75fab76665f46536cfc98ae9d892984edd64c Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 29 Aug 2018 13:17:50 +0800
Subject: [PATCH 4480/5242] MLK-19381 ARM64: dts: freescale: imx8qm: increase
 CMA size

commit  6632a0ad4d13b476d78e612457c1b820d00c94c0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Increase i.MX8QM CMA size to 960MB to meet 4K video requirement.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 23c91e8..6b17aa9 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -117,8 +117,8 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0 0x28000000>;
-			alloc-ranges = <0 0x96000000 0 0x28000000>;
+			size = <0 0x3c000000>;
+			alloc-ranges = <0 0x96000000 0 0x3c000000>;
 			linux,cma-default;
 		};
 
-- 
1.7.9.5

